app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt

```diff
@@ -140,6 +141,7 @@ private fun ThreePaneScaffoldNavigator<DestinationContent>.navigateToDetails(
     direction: TransferDirection,
     transferUuid: String,
 ) {
+    MatomoSwissTransfer.trackScreen("${direction.name.lowercase()}TransferDetailView")
```

Comment:
Maybe we should add a comment to the `TransferDirection` enum ?
Something like _"// Don't rename these, as it will break Matomo"_.

Or maybe even better, add a `matomoValue` parameter to this enum ?